### PR TITLE
Python3: port tests in eventsource

### PR DESCRIPTION
**Pull request by ziransun**

### CODE REVIEW

File: eventsource/resources/cors-cookie.py [hunk 1]
```diff
@@ -1,32 +1,33 @@
 from datetime import datetime
-from six import ensure_str
+
+from wptserve.utils import isomorphic_decode
```
Comment (jdm) [inline eventsource/resources/cors-cookie.py:3]: This is unused now.

File: eventsource/resources/last-event-id.py [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
 
-  last_event_id = ensure_str(request.headers.get("Last-Event-ID", ""))
+  last_event_id = request.headers.get(b"Last-Event-ID", b"")
   if last_event_id:
-    return "data: " + last_event_id + "\n\n"
+    return b"data: " + last_event_id + b"\n\n"
   else:
-    idvalue = request.GET.first("idvalue", u"\u2026")
-    return "id: " + idvalue + "\nretry: 200\ndata: hello\n\n"
+    idvalue = request.GET.first(b"idvalue", u"\u2026".encode("utf-8"))
```
Comment (jdm) [inline eventsource/resources/last-event-id.py:8]: Why not isomorphic_encode?

**Review by jdm — CHANGES_REQUESTED** [review]

File: eventsource/resources/cors-cookie.py (hunk 1 above)
Comment (ziransun) [reply eventsource/resources/cors-cookie.py]: Updated. Thank you!

File: eventsource/resources/last-event-id.py (hunk 2 above)
Comment (ziransun) [reply eventsource/resources/last-event-id.py:8]: This is similar to the discussion at 
https://github.com/web-platform-tests/wpt/pull/24091#discussion_r438367986

**Review by jdm — APPROVED** [review]

**Pull request merged**